scripts/operations/configuration/write_root_secrets_to_vault.py

```diff
@@ -38,105 +38,101 @@
 # eventually be implemented, but for the initial version of this tool, it is not
 # necessary.
 
-import base64
-import kubernetes
-import kubernetes.client
-import kubernetes.client.api
-import kubernetes.config
-import requests
+import argparse
+import logging
+import logging.config
 import sys
-import time
-
-from python_lib.common import err, err_exit, stdout_write_flush
-
-VAULT_SECRET_FIELD_NAMES = [ "password", "ssh_private_key", "ssh_public_key" ]
-
-def read_file(filename):
-    """
-    Reads contents of text file, verifies it is more than a few characters long,
-    and returns it as a string. The length check could be refined to be
-    more precise, but this will suffice.
-    """
-
-    print("Reading in file '%s'" % filename, flush=True)
-    try:
-        with open(filename, "rt") as textfile:
-            contents = textfile.read()
-    except FileNotFoundError:
-        err_exit("File '%s' does not exist" % filename)
-    if len(contents) < 4:
-        err_exit("File '%s' only contains %d characters" % (filename, len(contents)))
-    return contents
-
-def get_root_hash_from_etc_shadow():
+import traceback
+
+from python_lib import common
+from python_lib import vault
+
+class _ExcludeErrorsFilter(logging.Filter):
```

Comment:
Just as an aside (doesn't have to be done now) the logging config might be nice to factor into another module (e.g. `logging.py` or something) in the future.

Reply:
Completely agree.